src/components/form/form.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+/**
+ * Form Component using react-hook-form smart form compoment
+ * to inject all react-hook-form methods into the child components
+ * @author [Jake Bohn](https://github.com/JakobOrion)
+ */
+
+export default function Form({
```

Comment:
please change to` export const Form =({}) =>{...`

do everywhere
and on index.js `export * from "./forms"`